src/components/Bar.jsx

```diff
@@ -27,9 +26,9 @@ class Bar extends Component {
   }
 
   async componentWillMount () {
-    const appsList = await this.store.getAppsList()
+    await this.store.fetchAppsList()
```

Comment:
It just won't fetch the appsList in the store and you will have an empty apps list.